go/host/host.go

```diff
@@ -635,7 +635,7 @@ func (a *Node) processSharedSecretResponse(_ *ethadapter.L1RespondSecretTx) erro
 	var filteredHostAddresses []string //nolint:prealloc
 	for _, hostAddress := range hostAddresses {
 		// We exclude our own address.
-		if hostAddress == a.config.P2PAddress {
+		if hostAddress == a.config.P2PBindAddress {
```

Comment:
shouldn't this be the advertised address?

Reply:
Fixed ! :)